samples/sensor/sht3xd/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.8.2)
```

Comment:
Please add a license/copyright header.

@nashif Should the new github checks be catching this now?